src/nova_router.erl

```diff
@@ -113,6 +113,8 @@ compile([App|Tl], Dispatch, Options) ->
     %% Fetch the router-module for this application
     Router = erlang:list_to_atom(erlang:atom_to_list(App) ++ "_router"),
     Env = nova:get_environment(),
+    %% Ensure that the module is loaded
+    code:ensure_loaded(Router),
```

Comment:
This row doesn't do anything. we are not sure if the module is loaded or not, we need to have it in a case or match. 

{module, Router} = code:ensure_loaded(Router),

Reply:
The reason I added it without matching is cause it will crash either way. But thinking a bit of this I changed it in favour for erlang:apply/3 instead which works a lot better when using releases :)